statemachine/statemachine.py

```diff
@@ -80,8 +80,7 @@ def __repr__(self):
     def __or__(self, other):
         return CombinedTransition(self, other, identifier=self.identifier)
 
-    def __contribute_to_class__(self, managed, identifier):
-        self.managed = managed
+    def _set_identifier(self, identifier):
```

Comment:
`managed` was a reference to the statemachine class when the State or Transition was defined.  Removed as there's no real use-case and can generate conflict with a State now being used on multiple classes due to inheritance.